.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Thank you for reporting a possible bug in idb-connector.
+
+Please fill in as much of the template below as you can.
+
+Node.js version: output of `node -v`
+idb-connector version: output of `npm ls idb-connector`
+Platform: output of `uname -a`
+
+If possible, please provide code that demonstrates the problem, keeping it as
+simple and free of external dependencies as you can.
+
+And please provide the output with debugging messages -- `dbconn.debug(true)`
+-->
+
+* **Node.js version**:
+* **idb-connector version**:
+* **Platform**:
```

Comment:
Perhaps instead of "Platform" we use "Operating System" and ask the version as well.